common/src/main/java/org/cloudfoundry/identity/uaa/zone/IdentityZoneResolvingFilter.java

```diff
@@ -44,22 +43,28 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
         IdentityZone identityZone = null;
         String hostname = request.getServerName();
         String subdomain = getSubdomain(hostname);
+        logger.debug(String.format("hostname=[%s], subdomain=[%s]", hostname, subdomain));
+        
         if (subdomain != null) {
             try {
                 identityZone = dao.retrieveBySubdomain(subdomain);
             } catch (EmptyResultDataAccessException ex) {
-                logger.debug("Cannot find identity zone for subdomain " + subdomain, ex);
+                logger.info(String.format("Cannot find identity zone for subdomain %s", subdomain));
             } catch (Exception ex) {
-                logger.debug("Internal server error while fetching identity zone for subdomain" + subdomain, ex);
-                response.sendError(HttpServletResponse.SC_INTERNAL_SERVER_ERROR, "Internal server error while fetching identity zone for subdomain " + subdomain);
+            	String msg = String.format("Internal server error while fetching identity zone for subdomain %s", subdomain);
+                logger.info(msg, ex);
+                response.sendError(HttpServletResponse.SC_INTERNAL_SERVER_ERROR, msg);
                 return;
             }
         }
         if (identityZone == null) {
-            response.sendError(HttpServletResponse.SC_NOT_FOUND, "Cannot find identity zone for subdomain " + subdomain);
+        	String msg = String.format("Cannot find identity zone for subdomain %s", subdomain);
+        	logger.info(msg);
```

Comment:
This is an expected use case, there is no need to do any logging here. The only way to this code path, is logged above already.